packages/plugin-core/package.json

```diff
@@ -1,752 +1,757 @@
 {
-  "name": "@dendronhq/plugin-core",
```

Comment:
Oops - did we do some 'prettier' or whitespace formatting changes recently to this file? The only material difference should be the addition of 

```json
            {
                "command": "dendron.randomNote",
                "title": "Dendron: Random Note",
                "desc": "Open a Random Note"
            },
```

Reply:
yeah, we did some formatting. no worries about this

Reply:
is it ok to merge as is? or would you prefer me to re-align to the whitespace changes currently in master?

Reply:
can you update the package generation code? specifically [this](https://github.com/dendronhq/dendron/blob/dev-kevin/packages/plugin-core/scripts/genConfig.ts#L85:L85) should be changed to 2 spaces

Reply:
updated